VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -811,6 +799,15 @@ def get_workload_running(self):
         
     def set_pre_post_enabled(self):
         self.pre_post_enabled = True
+
+    def command_output_from_subprocess(self , args, process_wait_time):
```

Comment:
do we need try-catch?
or caller will handle any exceptions?

Reply:
It's better the caller function handles it. As we cannot suppress this error and we need to throw it so the caller function anyway needs to handle it.